### PR TITLE
github PR messaging

### PR DESCRIPTION
**Pull request by macdiesel**
@edx/pipeline-team PTAL.

@doctoryes I will rebase this after #94 lands.

### CODE REVIEW

File: tubular/release.py [hunk 1]
```diff
@@ -279,6 +290,98 @@ def batch(batchable):
...
+
+        Returns:
+            github.IssueComment.IssueComment
+
+        """
+        return self.message_pull_request(pr_number, PR_ON_STAGE_MESSAGE, force_message)
+
+    def message_pr_deployed_prod(self, pr_number, force_message=False):
+        """
+        sends a message that this PRs commits have been deployed to the produciton environment
```
Comment (doctoryes) [inline tubular/release.py]: Nit: typo

**Review by doctoryes — COMMENTED** [review]
Reviewed - but withholding approval until https://github.com/edx/tubular/pull/94 is merged. Hardball!

**Comment by macdiesel** [comment]
@doctoryes PR is rebased. I improved a few tests and fixed the typo. Please review.

**Review by doctoryes — APPROVED** [review]
🚢 🇮🇹

**Pull request merged**